spec/features/reviews/user_can_edit_review_spec.rb

```diff
@@ -80,19 +80,7 @@
       click_button 'Submit'
 
       expect(current_path).to eq("/shelters/#{@review1.id}/reviews/edit")
-      expect(page).to have_content("A required field is missing; please fill out all fields!")
-
-      fill_in 'Title', with: title2
-      fill_in 'Rating', with: rating2
-      fill_in 'Content', with: content2
-      fill_in 'Picture', with: ""
-      # user story does not mention picture as a required field, should it pass
-      # without flash message if there is no picture?
-
-      click_button 'Submit'
-
-      expect(current_path).to eq("/shelters/#{@review1.id}/reviews/edit")
-      expect(page).to have_content("A required field is missing; please fill out all fields!")
```

Comment:
Addressing your comment above; This test is not needed since the picture is optional and this would actually re-reroute them back to the shelter show page. If you want to be more robust, I am all for adding it back in, but we would need to change the expected path back to the show page.